src/libimg/Image.cpp

```diff
@@ -504,4 +727,16 @@ void Image::rotate(Image& dst, int angle) {
     }
 }
 
+void Image::drawText(const std::string text, int size, int x, int y, uint32_t color) {
+    // Centered around x,y
+    TTFStamper * name = new TTFStamper("Inconsolata.ttf");

```

Comment:
Initializing the `TTFStamper` is rather expensive because the TTF file needs to be loaded and parsed. Since the font is always the same, it would be less expensive to use a static instance throughout the `Image` class that can always be reused.
Also, `new` and `delete` should be avoided in modern C++. This could simply be replaced by `std::make_shared` which internally calls `new` and automatically frees the memory when the variable goes out of scope.